### PR TITLE
docs: add redis sd center and add sd config

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

docs

#### What this PR does / why we need it (en: English/zh: Chinese):

en: add redis sd center and add sd config.
zh: 新增 redis 服务注册发现中心，新增服务发现配置。

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: content/en/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 1]
```diff
@@ -17,6 +17,49 @@ As of now, the supported service discovery extensions are
 - [servicecomb](https://github.com/hertz-contrib/registry/tree/main/servicecomb)
 - [zookeeper](https://github.com/hertz-contrib/registry/tree/main/zookeeper)
 
+## Configuration
```
Comment (li-jin-gou) [inline content/en/docs/hertz/tutorials/service-governance/service_discovery.md:20]: add a table

**Comment by li-jin-gou** [comment]
PATL @justlorain

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 2]
```diff
@@ -1943,3 +1987,469 @@ func main() {
...
+            Tags:        nil,
+        }),
+    )
+    // ...
+}
+```
+
+##### WithWriteTimeout
+
+Redis 扩展提供了 `WithReadTimeout` 配置写入 socket 超时的时间，默认为 `ReadTimeout` 。
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2163]: `WithReadTimeout` -> `WithWriteTimeout`,
“默认为” -> “默认等同于”

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 3]
```diff
@@ -1943,3 +1987,469 @@ func main() {
...
+    )
+    // ...
+}
+```
+
+### 服务发现
+
+#### Option
+
+Etcd 拓展在服务发现部分中提供了 option 配置。
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2223]: “Etcd” -> "Redis"

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 4]
```diff
@@ -1943,3 +1987,469 @@ func main() {
...
+    // ...
+    ))
+    cli.Use(sd.Discovery(r))
+    // ...
+}
+```
+
+##### WithWriteTimeout
+
+Redis 扩展提供了 `WithReadTimeout` 配置写入 socket 超时的时间，默认为 `ReadTimeout` 。
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2343]: same with service registration

File: content/en/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 5]
```diff
@@ -1943,3 +1994,468 @@ The configuration of Zookeeper client and server can be customized, refer to the
...
+}
+```
+
+### Configuration
+
+You can customize the configuration of redis client and server, refer to [go-redis](https://github.com/go-redis/redis) configuration.
+
+### Complete Example
+
+For moer, see [example](https://github.com/hertz-contrib/registry/tree/main/redis/example).
```
Comment (justlorain) [inline content/en/docs/hertz/tutorials/service-governance/service_discovery.md:2461]: typo

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md (hunk 2 above)
Comment (justlorain) [reply content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2163]: Please also check the English doc, thx.

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md (hunk 3 above)
Comment (justlorain) [reply content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2223]: Please also check the English doc, thx.

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 6]
```diff
@@ -16,6 +16,57 @@ description: >
...
+`requestOptins` 提供 `WithTag` 配置项，使用此配置会为 `requestOptions` 设置 tag。
+
+示例代码：
+```go
+status, body, err := cli.Get(context.Background(), nil, "http://hertz.test.demo/ping", config.WithTag("foo", "var"))
+```
+
+### WithCustomizedAddrs
+
+此配置项指定服务发现时的目标实例地址。它将会覆盖来自 `Resolver` 的结果。
```
Comment (BaiZe1998) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:52]: Maybe we could make a brief introduction about `Resolver`.

File: content/en/docs/hertz/tutorials/service-governance/service_discovery.md (hunk 5 above)
Comment (L2ncE) [reply content/en/docs/hertz/tutorials/service-governance/service_discovery.md]: All problems solved, thanks.

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 7]
```diff
@@ -1943,3 +1994,469 @@ func main() {
...
+
+```go
+go get github.com/hertz-contrib/registry/redis
+```
+
+### 服务注册
+
+#### Option
+
+Redis 拓展在服务注册部分中提供了 option 配置。
```
Comment (BaiZe1998) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2010]: Add a table for configuration list.

Comment (L2ncE) [reply content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2010]: > Add a table for configuration list.

The table is not provided here to match the previous document.

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md (hunk 6 above)
Comment (L2ncE) [reply content/zh/docs/hertz/tutorials/service-governance/service_discovery.md]: done, thx.

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 8]
```diff
@@ -1943,3 +1994,469 @@ func main() {
...
+            Tags:        nil,
+        }),
+    )
+    // ...
+}
+```
+
+##### WithWriteTimeout
+
+Redis 扩展提供了 `WithReadTimeout` 配置写入 socket 超时的时间，默认等同于 `ReadTimeout` 。
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2163]: `WithReadTimeout` -> `WithWriteTimeout`

File: content/zh/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 9]
```diff
@@ -1943,3 +1994,469 @@ func main() {
...
+    // ...
+    ))
+    cli.Use(sd.Discovery(r))
+    // ...
+}
+```
+
+##### WithWriteTimeout
+
+Redis 扩展提供了 `WithReadTimeout` 配置写入 socket 超时的时间，默认等同于 `ReadTimeout` 。
```
Comment (justlorain) [inline content/zh/docs/hertz/tutorials/service-governance/service_discovery.md:2343]: `WithReadTimeout` -> `WithWriteTimeout`

File: content/en/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 10]
```diff
@@ -1943,3 +1994,468 @@ The configuration of Zookeeper client and server can be customized, refer to the
...
+             Tags: nil,
+         }),
+     )
+     //...
+}
+```
+
+##### WithWriteTimeout
+
+The Redis extension provides `WithReadTimeout` to configure the write socket timeout time, the default is equivalent to `ReadTimeout`.
```
Comment (justlorain) [inline content/en/docs/hertz/tutorials/service-governance/service_discovery.md:2163]: `WithReadTimeout` -> `WithWriteTimeout`

File: content/en/docs/hertz/tutorials/service-governance/service_discovery.md [hunk 11]
```diff
@@ -1943,3 +1994,468 @@ The configuration of Zookeeper client and server can be customized, refer to the
...
+     //...
+     ))
+     cli. Use(sd. Discovery(r))
+     //...
+}
+```
+
+##### WithWriteTimeout
+
+The Redis extension provides `WithReadTimeout` to configure the write socket timeout time, the default is equivalent to `ReadTimeout`.
```
Comment (justlorain) [inline content/en/docs/hertz/tutorials/service-governance/service_discovery.md:2342]: `WithReadTimeout` -> `WithWriteTimeout`

Comment (L2ncE) [reply content/en/docs/hertz/tutorials/service-governance/service_discovery.md]: > `WithReadTimeout` -> `WithWriteTimeout`

All fixed, thanks.

**Review by justlorain — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**